utility/BUILD.bazel

```diff
@@ -73,9 +73,26 @@ kt_android_library(
     ],
 )
 
+# Place your file here if:
+#  - It is not a test file
+#  - Only test files depend on it
+
+kt_android_library(
+    name = "test_deps",
+    testonly = True,
+    srcs = [
+        "src/test/java/org/oppia/android/util/utility/NetworkConnectionTestUtil.kt",
+    ],
+    deps = [
+        "//third_party:javax_inject_javax_inject",
+        "//third_party:org_robolectric_robolectric",
+    ],
+)
```

Comment:
Instead of doing this, we should have a network_connection_test_util library within src/main/java/org/oppia/android/util/networking/BUILD.bazel that's testonly & used by tests.

Note that test utilities go in **main** not **test**. ``test`` is only for tests.

Reply:
@yashraj-01 was this resolved?

Reply:
Actually, the `NetworkConnectionTetsUtil` imports Shadow from robolectric and when I put it in **main**, the import statement was showing an error. It was not able to identify robolectric. So for now, I have moved it to test/networking. Is it okay here? If not, then how can we resolve this error?
![Screenshot from 2021-08-05 00-44-51](https://user-images.githubusercontent.com/63469151/128241300-7a0a361c-385a-4307-8dbd-c28c3c0d9d1b.png)

Reply:
Aha. That's unfortunate (it's another limitation of Gradle unfortunately). One thing you could do is move the utility & its test to the testing package (e.g. org.oppia.android.testing.networking) since that module is allowed to have non-test files that specifically depend on Robolectric.

Reply:
Moved to testing/networking package